erpnext/projects/doctype/project/project.py

```diff
@@ -79,6 +79,25 @@ def validate_project_name(self):
 			frappe.throw(_("Project {0} already exists").format(frappe.safe_decode(self.project_name)))
 
 	def validate_dates(self):
+		if self.tasks:
+			for d in self.tasks:
+				if self.expected_start_date:
+					if d.start_date and getdate(d.start_date) < getdate(self.expected_start_date):
+						frappe.throw(_("Start date of task <b>{0}</b> cannot be less than <b>{1}</b> expected start date")
```

Comment:
Show the actual End Date of the project in this message.

Start date of task **ABC** cannot be less than project's start date **2019-09-08**

Reply:
@netchampfaris Done